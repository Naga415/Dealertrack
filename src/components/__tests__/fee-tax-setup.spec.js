import React from "react";
import renderer from "react-test-renderer";
import FeeTaxSetup from "../fee-tax-setup";

describe("Fee Tax Setup specs", () => {
  const props = {
    htmlId: "FeeTaxSetup",
    data: [{ feeType: "test1" }, { feeType: "test2" }],
    actions: {}
  };

  it("renders <FeeTaxSetup />", () => {
    const component = renderer.create(<FeeTaxSetup {...props} />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
