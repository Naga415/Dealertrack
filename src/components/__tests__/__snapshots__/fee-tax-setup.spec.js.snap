// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fee Tax Setup specs renders <FeeTaxSetup /> 1`] = `
<div
  className="row"
  id="edit-tax-fee-setup"
>
  <div
    className="nav-tabs-content"
  >
    <strong
      className="text"
    >
      Following are the fees available.
    </strong>
    <table
      className="table fee-tax-setup-table"
    >
      <thead>
        <tr>
          <th>
            Fee Type
          </th>
          <th>
            Fee Name
          </th>
          <th>
            Tax if cap
          </th>
          <th>
            Tax if DIA
          </th>
          <th>
            CRT
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            test1
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
        </tr>
        <tr>
          <td>
            test2
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
          <td>
            <input
              type="checkbox"
            />
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
